packages/components/Button/package.json

```diff
@@ -28,19 +28,19 @@
     "@fluentui-react-native/pressable": ">=0.5.1 <1.0.0",
     "@fluentui-react-native/text": ">=0.8.2 <1.0.0",
     "@fluentui-react-native/tokens": ">=0.7.1 <1.0.0",
-    "@office-iss/react-native-win32": "0.62.6",
     "@uifabricshared/foundation-composable": ">=0.7.0 <1.0.0",
     "@uifabricshared/foundation-settings": ">=0.8.0 <1.0.0"
   },
   "devDependencies": {
-    "@types/react-native": "^0.62.0",
+    "@office-iss/react-native-win32": "^0.63.7",
+    "@types/react-native": "^0.63.0",
     "@uifabricshared/build-native": "^0.1.1",
     "@uifabricshared/eslint-config-rules": "^0.1.1",
-    "react-native": "0.62.2"
+    "react-native": "^0.63.4"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-native": ">=0.60.0"
+    "react": ">=16.13.1",
+    "react-native": ">=0.63.4"
```

Comment:
Do we need to specify it as both a devDependency and peerDependency? My worry is that having it a devDependency increases the likelihood the monorepo can pull in two versions of react native that don't play nice.

My last thought on this was to have every package (Excpt the test apps) specify them as peer dependencies, then use [yarn resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) to specify one version of react-native/react-native-macos/windows/win32 for the whole monorepo.